python/benchmarks/parse_pet.py

```diff
@@ -171,3 +183,24 @@ def _get_xml(uri):
             return dd
     except Exception:
         return {}
+
+
+@click.command
+@click.option("-u", "--base-uri", type=str, required=True, help="Oxford Pet dataset root")
+@click.option("-f", "--fmt", type=click.Choice(["parquet", "lance"]), help="Output format (parquet or lance)")
+@click.option("-o", "--output", type=str, default="oxford_pet.lance", help="Output path")
+def main(base_uri, fmt, output):
```

Comment:
Would it make sense to roll-up the main methods into the super class (like with the benchmarks)?